openedx/core/djangoapps/appsembler/settings/settings/production_common.py

```diff
@@ -61,7 +61,7 @@ def plugin_settings(settings):
         settings.TIERS_ORGANIZATION_TIER_GETTER_NAME = 'get_tier_for_org'
 
         settings.TIERS_DATABASE_URL = settings.AUTH_TOKENS.get('TIERS_DATABASE_URL')
-        settings.DATABASES['tiers'] = dj_database_url.parse(settings.TIERS_DATABASE_URL)
+        settings.DATABASES['tiers'] = dj_database_url.parse(settings.TIERS_DATABASE_URL, ssl_require=True)
```

Comment:
`production_common.py` may run in `devstack` for weird edX platoform reasons.

It's better to avoid such issues and allow it to be configured:

```suggestion
        settings.DATABASES['tiers'] = dj_database_url.parse(settings.TIERS_DATABASE_URL, 
                                                                                              ssl_require=settings.AUTH_TOKENS.get('TIERS_REQUIRE_SSL', True))
```